lib/umwelt/parser/def.ex

```diff
@@ -0,0 +1,19 @@
+defmodule Umwelt.Parser.Def do
+  @moduledoc "Parses Method AST"
+
+  alias Umwelt.Parser
+
+  def parse({:def, _, [{:when, _, _} = ast, [do: _]]}, aliases),
+    do: Parser.When.parse(ast, aliases)
+
+  def parse({:def, _, [{method, _, arguments}, [do: _]]}, aliases) do
+    %{}
+    |> Map.put(:method, method)
```

Comment:
```suggestion
    |> Map.put(:function, method)
```

Calling a function “method” is a sin in _elixir_ :-P